R/build_lookup.R

```diff
@@ -39,23 +62,36 @@ extract_imgt_genes <- function(data_dir) {
 #' @param gene_str A string, the gene name.
 #'
 #' @return A string, the updated gene name.
+#' @export
+#' @keywords internal
+#' @examples
+#' add_dash_one('TRBV2*01')
 add_dash_one <- function(gene_str) {
+  if (!length(gene_str) == 1) {
```

Comment:
beware with differences between R and python for string length. `length("abc") == 1`  in R and `len("abc") == 3` in Python

Reply:
if you want the number of characters in a string, do `nchar()`. 

length does that in R i think because R is vectorized whenever possible, so length of a vector of strings is just the number of strings in the vector `length(letters) == 26`

Reply:
maybe you do want to make sure it's only 1 string in which case what you have is great

Reply:
Yes I do just want to check it's one string in this case, but I've been burned by the `length()` `len()` difference recently so thanks for bringing it up!